index.html

```diff
@@ -0,0 +1,484 @@
+          </g>
+        </svg>
+        <h1 class="f-light">Hi there! We're Avengers, <strong>creative design agency</strong> we build <strong><span class="underline">awesome</span></strong> mobile and websites</h1>
+      </section>
+      <article class="b-orange" id="services">
+        <h2 class="f-bold f-large">SERVICES</h2>
+        <section class="l-flex-container">
+          <p class="f-font-paragraph l-flex-item l-svcs-txt-a">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+          <p class="f-font-paragraph l-flex-item l-svcs-txt-b">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+          <p class="f-font-paragraph l-flex-item l-svcs-txt-c">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+        </section>
+        <section>
+          <div class="l-flex-container l-svcs-txt">
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">UI & UX DESIGN</h3>
+              <span class="icon-brush icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">LABORATORY</h3>
+              <span class="icon-lab icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">DEVELOPMENT</h3>
+              <span class="icon-cog icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+          </div>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">Pricing Tables</h3>
+          <div class="l-flex-container">
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">BASIC</th>
```

Comment:
Is the class of `.th-title` really necessary? Or could targeting `.l-flex-prices th` in your CSS be sufficient? A `th` is, by definition, a table heading and thus an implied title, so the classname is redundant.